charts/influxdb2/Chart.yaml

```diff
@@ -5,7 +5,7 @@ name: influxdb2
 description: A Helm chart for InfluxDB v2
 home: https://www.influxdata.com/products/influxdb-overview/influxdb-2-0/
 type: application
-version: 1.0.17
+version: 2.0.0
```

Comment:
Since there's no `2.0.0` on DockerHub, should this be `2.0` or `2.0.4`? That way the fallback case will work.